xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -2778,11 +2778,13 @@ finally {
         <Parameter Name="value" Type="TValue" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="key">The key of the element to add.</param>
+        <param name="value">The value of the element to add. It can be <see langword="null" />.</param>
+        <summary>Attempts to add the specified key and value to the dictionary.</summary>
+        <returns><see langword="true" /> if the key/value pair was added to the dictionary successfully; otherwise, <see langword="false" />.</returns>
+        <remarks>Unlike the <see cref="M:System.Collections.Generic.Dictionary`2.Add(`0,`1)" /> method, this method doesn't throw an exception if the element with the given key exists in the dictionary. Unlike the Dictionary indexer, TryAdd doesn't override the element if the element with the given key exists in the dictionary. If the key already exists, TryAdd does nothing and returns <see langword="false" />.</remarks>
```

Comment:
Ok, the build didn't link my method link for some reason. Let's try to switch it to Markdown instead as we usually do for Remarks.

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  

Unlike the <xref:System.Collections.Generic.Dictionary%602.Add%2A" /> method, this method doesn't throw an exception if the element with the given key exists in the dictionary. Unlike the Dictionary indexer, `TryAdd` doesn't override the element if the element with the given key exists in the dictionary. If the key already exists, `TryAdd` does nothing and returns `false`.

 ]]></format>
        </remarks>
```